src/proteingym/base/dataset.py

```diff
@@ -322,6 +367,21 @@ def _validate_msa_reference_sequences(self) -> "Dataset":
                 )
         return self
 
+    def fill_from_database(self) -> "Dataset":
```

Comment:
So not "in-place" - also makes sense but should be consistent with other methods then